DMCompiler/DMStandard/Types/Client.dm

```diff
@@ -2,6 +2,7 @@
 	var/list/verbs = list()
 	var/list/screen = list()
 	var/list/images = list() as opendream_unimplemented
+	var/list/vars
```

Comment:
Is setting `/client/parent_type = /datum` going to cause issues with duplicate vars?

Reply:
Yuhp. Only solution I can think of would be some snowflake thing in the duplicate var detection.